_posts/brainhackGlobal2020.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Brainhack Global 2020"
+date:   2020-11-30 13:53
+project_categories: Events 2020
+image: bhack_global_2019.png
```

Comment:
This should be the name of the 2020 edition. Please name it as `bhack_global_2020.png` so that I can add in a separate PR.